esterni/manuale-utente/components/mobile-application.tex

```diff
@@ -0,0 +1,134 @@
+In caso contrario, se l'utente inserisce delle credenziali non valide, l'accesso a Stalker sarà per forza negato.\\
+Le cause possono essere le seguenti:
+\begin{itemize}
+    \item l'email inserita non rispetta i vincoli imposti dal sistema di Stalker.
+    \item la password non rispetta i vincoli imposti dal sistema di Stalker.
+    \item l'email e la password inserite non rispettano i vincoli imposti dal sistema di Stalker.
+    \item l'email inserita rispetta i vincoli, ma non è presente all'interno del database di Stalker.
+    \item la password inserita rispetta i vincoli, ma non è presente all'interno del database di Stalker.
+    \item l'email e la password inserite rispettano i vincoli, ma non sono presenti all'interno del database di Stalker.
+\end{itemize}
+
+Se l'utente non possiede delle credenziali, ha la possibilità di registrarsi, grazie all'apposito form disponibile dopo aver cliccato su \textit{Not yet registered? Sign up}.
+\newpage
+
+\subsubsection{Pagina di registrazione}%
+\label{sub:pagina_di_registrazione}
+
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=70mm]{img/mobile-app/pagina-di-registrazione.jpg}
+    \caption{Pagina di registrazione}%
+    \label{fig:mobile_app_pagina_di_registrazione}
+\end{figure}
+
+Se l'utente non si è mai registrato in Stalker, questo form è l'unico modo per farlo!\\
+L'utente per registrarsi deve inserire delle semplici informazioni personali:
+
+\begin{itemize}
+    \item inserire il nome.
+    \item inserire il cognome.
+    \item inserire la data di nascita.
+    \item inserire l'email, non precedentemente utilizzata in Stalker.
+    \item inserire la password, che deve essere alfanumerica con almeno 8 caratteri e al massimo 32.
+    \item premere il pulsante sign up per effettuare la registrazione.
+\end{itemize}
+
+Si avvisa inoltre che non è possibile registrarsi più volte con la stessa email.\\
+Se questi vincoli non saranno rispettati l'utente non potrà registrarsi.\\
+Se l'utente possiede delle credenziali, ha la possibilità di accedere, grazie all'apposito form disponibile dopo aver cliccato su \textit{Already have an account? Login}.
+\newpage
+
+\subsubsection{Pagina iniziale}%
+\label{sub:pagina_iniziale}
+
+% \begin{figure}[H]
+%     \centering
+%     \includegraphics{img/mobile-app/pagina-iniziale.png}
+%     \caption{Pagina iniziale}%
+%     \label{fig:mobile_app_pagina_iniziale}
+% \end{figure}
+
+Lorem ipsum
+
+% add other functionalities 
+\newpage
+\subsection{Risoluzione dei problemi}
+
+\subsubsection{Accesso all'applicazione non disponibile}
+
+Nel caso ci fossero problemi con il collegamento all'app, vi consigliamo di controllare se al momento è attiva una rete internet nel vostro smartphone Android.
+Nel caso in cui la rete sia abilitata vi preghiamo di riprovare l’accesso in un secondo momento.
+Se il problema dovesse persistere vi preghiamo di segnalarcelo nelle modalità discusse nella sezione \emph{Supporto tecnico}.
```

Comment:
Non so se sarebbe meglio usare una forma più impersonale dei verbi (e questo vale per tutta la sezione "Risoluzione dei problemi", non solo per questi paragrafi), meglio chiedere a @RiccardoAgatea 

Tipo: "vi consigliamo" -> "è consigliato"

In ogni caso direi di usare **vi** al posto che **ci**: "Nel caso vi fossero problemi [...]"

Reply:
Come ho scritto più in alto, sta cosa di forma attiva vs impersonale andrebbe chiesta al prof. 

La differenza fra "vi" e "ci" (in questo caso) è marginale.

Reply:
Ok, quindi al momento teniamo così.